capstone/pom.xml

```diff
@@ -119,19 +119,6 @@
       <artifactId>appengine-remote-api</artifactId>
       <version>1.9.64</version>
     </dependency>
- 
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-web</artifactId>
-      <version>5.2.8.RELEASE</version>
-    </dependency>
- 
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-test</artifactId>
-      <version>5.2.8.RELEASE</version>
-      <scope>test</scope>
-    </dependency>
```

Comment:
I think we actually need these dependencies...they got removed accidentally in another pull request but should be there.

Reply:
Josh did you sync with Katie offline or does this comment still need to be addressed?

Reply:
There are two sets of those dependencies, if I'm not mistaken.